docs/service-discovery.md

```diff
@@ -73,27 +80,27 @@ To use DNS SD, just add one of the following prefixes to the domain name in your
 * `dns+` - the domain name after this prefix will be looked up as an A/AAAA query. *A port is required for this query type*. An example using this lookup with a static flag:
 
 ```
---store=dns+stores.thanos.mycompany.org:9090
+--endpoint.sd-config=endpoints: [{address: "dns+stores.thanos.mycompany.org:9090"}]
```

Comment:
Hmm maybe this should be quotes.

Also the value doesn't seem like valid YAML to me

Reply:
this works in k8s, couldn't test locally
```
    spec:
      containers:
      - args:
        - query
        - '--endpoint.sd-config=endpoints: [{address: ''prometheus-operated:10901''}]'
```